init-tools.cmd

```diff
@@ -61,11 +61,17 @@ echo Running: "%BUILD_TOOLS_PATH%init-tools.cmd" "%~dp0" "%DOTNET_CMD%" "%TOOLRU
 call "%BUILD_TOOLS_PATH%init-tools.cmd" "%~dp0" "%DOTNET_CMD%" "%TOOLRUNTIME_DIR%" >> "%INIT_TOOLS_LOG%"
 set INIT_TOOLS_ERRORLEVEL=%ERRORLEVEL%
 if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
-  echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details. 1>&2
+  echo ERROR: An error occured when trying to initialize the tools. Please check the detailed log that follows. 1>&2
+  type "%INIT_TOOLS_LOG%" 1>&2
```

Comment:
You might consider just having the body here jump to `:error` like elsewhere, instead of explicitly printing out the log.  I understand in this case it's possible that we might return a different error code in the failure case (if %ERRORLEVEL% is not equal to 1) but I don't think it's that big a deal.  Your call.